### PR TITLE
Story/ccls 2165 clamav

### PR DESCRIPTION
**Pull request by porritta**

### CODE REVIEW

File: src/integrationTest/resources/application.yml [hunk 1]
```diff
@@ -48,3 +48,10 @@ os:
     port: 8126
     url: ${os.api.hostname}:${os.api.port}
     key: key
+
+av:
+  enabled: false
+  api:
+    hostname: localhost
+    port: 8128
+    timeout: 20000
```
Comment (farrell-m) [inline src/integrationTest/resources/application.yml:57]: Are these required if av isn't enabled?

**Review by farrell-m — APPROVED** [review]
Looks good 👍 
1 small comment but mainly cosmetic

File: src/integrationTest/resources/application.yml (hunk 1 above)
Comment (porritta) [reply src/integrationTest/resources/application.yml:57]: Probably not - but the ClamAVClient is constructed whether or not AV is enabled.
It's a runtime call to the AvScanService.scan() method that checks the enabled flag before calling the external service.

Comment (porritta) [reply src/integrationTest/resources/application.yml:57]: The integration tests fail if either the api section is missing, or the hostname etc are blank.

**Pull request merged**